core/kernel/ldelf_loader.c

```diff
@@ -452,5 +496,10 @@ TEE_Result ldelf_dlsym(struct user_mode_ctx *uctx, TEE_UUID *uuid,
 			*val = arg->dlsym.val;
```

Comment:
`PUT_USER()` for `*val = arg->dlsym.val;` ?
`

Reply:
so I would prefer you add an inline comment stating the `PUT_USER()` is not needed here for that reasonb, otherwise it looks as if we missed something.